tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -68,7 +67,16 @@ def test_upgrade_ocp(self):
             ocp.upgrade_ocp(image=target_image, image_path=image_path)
 
             # Wait for upgrade
-            for ocp_operator in self.cluster_operators:
+            for ocp_operator in cluster_operators:
+                logger.info(f"Checking upgrade status of {ocp_operator}:")
+                # ############ Workaround for issue 2624 #######
```

Comment:
do we need a bug for https://github.com/red-hat-storage/ocs-ci/issues/2624?

Reply:
i dont think so, they just change the name of 2 cluster operators between versions, but it still same amount and system reports no errors